Utilities/WaveFrontReader.h

```diff
@@ -509,6 +509,12 @@ class WaveFrontReader
                 LoadTexturePath(InFile, curMaterial->strEmissiveTexture, MAX_PATH);
                 curMaterial->bEmissive = true;
             }
+            else if (0 == wcscmp(strCommand.c_str(), L"map_RMA")
```

Comment:
These are obviously more WaveFront OBJ extensions, and they don't appear in the [Wikipedia](https://en.wikipedia.org/wiki/Wavefront_.obj_file) article for the format. What toolchain is using these?

Also, keep in mind that channel order can vary here. For the _DirectX Tool Kit_ PBR pipeline, I assume the RMA texture is in glTF2 channel order:  The metalness is in the B channel, roughness in the G channel, and ambient occlusion in the R channel. 

Does whatever toolchain that produces these have a proscribed  RMA channel order?

Reply:
I'm unaware of any toolchain that embeds an RMA texture, as far as I know there is (of current) no standard command for one in MTL. Just seems like something that should be able to be defined per material rather than assuming the name based on the albeto name.

I added this method into our team's pipeline and it was useful, so figured I'd propose it.

Reply:
There was [one proposed](http://exocortex.com/blog/extending_wavefront_mtl_to_support_pbr) which is ``map_Pr``, ``map_Pm``, and ``map_Ps`` which requires extra processing to merge the channels into the expect glTF2 RGB form. That's why I didn't use it.

Reply:
Yeah fair point, up to you if you accept the PR. 

Personally I think it's an improvement - and definitely better than nothing. If the command was added to the docs it would give people the option to leverage it in future implementations.